hishel/_s3.py

```diff
@@ -1,14 +1,24 @@
+import time
 import typing as tp
 from datetime import datetime, timedelta, timezone
 
 from anyio import to_thread
 
+try:
+    from botocore.exceptions import ClientError
+except ImportError:  # pragma: no cover
+    ClientError = Exception  # type: ignore
```

Comment:
When do we need this?

Reply:
I guess we don't need this because of 
```python
if boto3 is None:  # pragma: no cover
            raise RuntimeError(
                (
                    f"The `{type(self).__name__}` was used, but the required packages were not found. "
                    "Check that you have `Hishel` installed with the `s3` extension as shown.\n"
                    "```pip install hishel[s3]```"
                )
            )
```

But you didn't import any boto3 classes and `client: tp.Any` so I thought there should be a reason for this.

Reply:
Let's then remove that part, and I think we can merge this

Reply:
> Let's then remove that part, and I think we can merge this

Got it 👍